app/views/collections/index.html.erb

```diff
@@ -55,3 +56,6 @@
     </a>
   <% end %>
 </div>
+<% else  %>
+<h1>Oops, You Must Login To See Your Collections</h1>
+<% end %>
```

Comment:
Glad we have this! Maybe we should reword it to also address those who do not have a username?